x-pack/functionbeat/manager/aws/event_stack_poller.go

```diff
@@ -124,7 +124,7 @@ func (e *eventStackPoller) poll() {
 			if nextToken == nil {
 				return
 			}
-		case <-time.After(periodicCheck):
+		case <-time.After(e.periodicCheck):
```

Comment:
Good catch, I think there is another variable with the same name in the package, this would explain why it did not fail at compile time.

Reply:
Yes. It was not an issue at all, because the same `periodicCheck` is just passed to `makeEventStackPoller` call which was used in the function. So the value is the same, it just looks correct now.